libarchive/archive_write_set_format_pax.c

```diff
@@ -1387,7 +1394,7 @@ archive_write_pax_header(struct archive_write *a,
 	 * numeric fields, though they're less critical.
 	 */
 	if (__archive_write_format_header_ustar(a, ustarbuff, entry_main, -1, 0,
-	    NULL) == ARCHIVE_FATAL) {
+	    sconv) == ARCHIVE_FATAL) {
```

Comment:
Oh, this looks to be somewhat annoying. From what I can tell, the pathname for `entry_main` is translated into target character set before this point in time, but the link path(s) - which this is trying to fix - is not :/